### DISCUSSION TITLE
How do you prevent someone from using their GitHub account to sign in to Outstatic?

### DISCUSSION THREAD

**Discussion by ZiyadBoshima** [avitorio/outstatic#136] [Q&A]
I noticed that I can use a different GitHub account to sign in to Outstatic. Is there a way I can limit sign in to only a specified Github user?

**Comment by avitorio** [comment]
Hey @ZiyadBoshima,

Can you give me some more info on the issue you are having?
Because, as it stands, other users would only be able to access the dashboard if you add them as collaborators to your repository.

**Reply by ZiyadBoshima** [reply]
So like users can sign in to the Outstatic page on my hosted website, and cam have access to the dashboard. Now they only have access to the dashboard, but not access to the data, meaning they cannot post or modify content on my GitHub repo. I assume this is because when you sign in, and authorize Outstatic, it would simply work with your own GitHub repos. Does that make sense?

Is there a way to limit user sign in, so that if it is not me that is signing in, Outstatic should not even allow for users to get to the dashboard? This is simply a concern to do with minimizing unnecessary access to admin pages. If you wish to test this, you can head over to my website (available in my profile page) and try to sign in.

**Reply by avitorio** [reply]
Ah, I see what you mean. People can sign in but they won't be able to see or edit your data.
There's currently no check to see if the person is a collaborator to your repo and not show the dashboard in that case.
I think this could be an easy check. Would you like to try implementing it?

We are also moving away from Next.js 12, so if you want to work on a new feature use the `canary` branch.

**Reply by ZiyadBoshima** [reply]
Yeah, I'll try implementing it.
